docs/mkdocs.yml

```diff
@@ -74,6 +74,7 @@ nav:
   - Experiment:
     - Defining an Experiment: experiments/defining.md
     - Testing Feature Configuration: experiments/feature-testing.md
+    - Experiment Configuration: example/conifg/README.md
```

Comment:
I don't think this is going to work: the root of the mkdocs search tree is `docs/sources` and I don't believe relative paths are allowed. You can use URLs, though, so you can link to the file on GIthub: `https://github.com/dssg/triage/blob/master/example/config/README.md` (this won't work until the branch is merged, of course)

Reply:
oh it was different path.